projects/github-actions/required-review/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.2.0] - 2023-09-12
+### Fixed
+- Replaced hardcoded values of github.com with variables to support GitHub Enterprise. [#32974]
+
```

Comment:
As above, don't do this, add a file in `projects/github-actions/required-review/changelog/` instead.